src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -43,6 +43,15 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
   private readonly zones: Zone[];
   private readonly uuidGenerator = new UuidGenerator();
 
+  private queuedCFChanges: Record<
```

Comment:
NIT: I would rename "queued..." into "pending...", in my head, queuing means delay with a certain order, in this case we delay but the order do not matter.

Reply:
Already named `queuedCFChanges` in 18+ :x